build.sbt

```diff
@@ -645,8 +659,13 @@ lazy val testRunner =
     .settings(mavenPublishSettings)
     .settings(
       crossScalaVersions := Seq(sbt10ScalaVersion),
-      libraryDependencies += "org.scala-sbt" % "test-interface" % "1.0",
-      Compile / sources ++= (testInterfaceSerialization / Compile / sources).value
+      libraryDependencies ++= Seq(
+        "org.scala-sbt" % "test-interface"  % "1.0",
+        "com.novocode"  % "junit-interface" % "0.11" % "test"
+      ),
+      Compile / sources ++= (testInterfaceCommon / Compile / sources).value,
+      Test / sources ++= (testInterfaceCommon / Test / sources).value,
+      Test / sources ++= (junitAsyncJVM / Compile / sources).value
```

Comment:
I've tried to change it, but then sbt cannot resolve dependency of `org/scala-native/junit-async-jvm_2.12/0.4.0-SNAPSHOT/junit-async-jvm_2.12-0.4.0-SNAPSHOT.pom`, do you know why this may happend? 
It's strange as it occurs in `test-runner`, but not in `junitTestOutputsJVM`. Both of them have dependency of `junitAsyncJvm`

Is it possible that this problem not exists in Scala.js as there is used MultiProject?

Reply:
Ah it's probably because junitAsyncJVM` is configured with Scala 2.11 for some reason? It should probably be configured with
```scala
scalaVersion := sbt10ScalaVersion
```
(as well as projects that depend on it)